### PR TITLE
Bugfix: Fix ProductCard Save.jsx

### PR DESCRIPTION
**Pull request by Mwindo**

### CODE REVIEW

File: affiliate/blocks/productCard/src/Save.jsx [hunk 1]
```diff
@@ -1,14 +1,13 @@
 import { useBlockProps } from '@wordpress/block-editor';
 
-import ProductCarousel from './ProductCard';
```
Comment (Mwindo) [inline affiliate/blocks/productCard/src/Save.jsx:3]: Well, this certainly won't do. I modified this file and forgot to push the change in my previous PR.

**Comment by Mwindo** [comment]
Going to merge and release this since it fixes something clearly broken, but happy to get a review afterwards.

File: affiliate/blocks/productCard/src/Save.jsx (hunk 1 above)
Comment (Mwindo) [reply affiliate/blocks/productCard/src/Save.jsx:3]: I've been testing on a feature branch, so nothing is broken in production.

**Review by amittman — COMMENTED** [review]
Ah, looks good. Unfortunate we didn't catch this with the previous. Should be able to set up tests for this branch this sprint.

**Comment by Mwindo** [comment]
> Should be able to set up tests for this branch this sprint.

I was thinking the same thing. It will be super helpful.

**Pull request merged**